monitoring/terraform/stack/grafana/load_balancer.tf

```diff
@@ -1,29 +1,47 @@
 resource "aws_alb" "alb" {
   # This name can only contain alphanumerics and hyphens
-  name = "${replace("${var.namespace}", "_", "-")}"
+  name = replace(var.namespace, "_", "-")
 
-  subnets = ["${var.public_subnets}"]
+  subnets = var.public_subnets
 
   security_groups = [
-    "${aws_security_group.service_lb_security_group.id}",
-    "${aws_security_group.external_lb_security_group.id}",
+    aws_security_group.service_lb_security_group.id,
+    aws_security_group.external_lb_security_group.id,
   ]
 }
 
+resource "aws_alb_target_group" "ecs_service" {
+  # We use snake case in a lot of places, but ALB Target Group names can
+  # only contain alphanumerics and hyphens.
+  name = replace(var.namespace, "_", "-")
+
+  target_type = "ip"
+
+  protocol = "HTTP"
+  port     = local.container_port
+  vpc_id   = var.vpc_id
+
+  health_check {
+    protocol = "HTTP"
+    path     = "/api/health"
```

Comment:
I can't see this in the previous tf, will it pass?

Reply:
[Wait, just seen it](https://github.com/wellcomecollection/platform/pull/4364/files#diff-e775a6b9b2fddd9eb9b4c0c846574c22)

Reply:
It’s copied directly from the previous TF.